brendenwest/cascade
# Individual Ride Screen

## Description
* We had an unused api for details on individual rides
* This PR implements the Ride Details Screen and stack navigation from the Featured Rides Flatlist to the relevant Details Screen by passing url props. 

See Issue #48 

## To Test
1.run npm install if needed
2. fire up Metro and run `npx react-native run-android`
3. Click onto Daily Featured Rides button on the home screen
4. Observe the loaded daily featured rides
5. Click on any ride in the list and observe the detail page load
6. Test out the register and external rides (ridewithgps.com) detail links
7. Look through the entire app to ensure no features were lost in rebase

*Note: the screen's datetime strings were consistently clean, but some were noticed having "add to calendar" trailing language. It may be a good ask for Brenden to clean up the api for consistency,
